src/istio/control/http/attributes_builder_test.cc

```diff
@@ -627,10 +627,10 @@ TEST(AttributesBuilderTest, TestReportAttributesWithDestIP) {
         info->response_flags = "NR";
       }));
   EXPECT_CALL(mock_data, GetGrpcStatus(_)).WillOnce(testing::Return(false));
-  EXPECT_CALL(mock_data, GetRBACPermissiveAttributes(_, _))
-      .WillOnce(Invoke([](std::string *code, std::string *id) {
-        *code = "403";
-        *id = "policy-foo";
+  EXPECT_CALL(mock_data, GetRbacReportInfo(_))
+      .WillOnce(Invoke([](ReportData::RbacReportInfo *rbacReportInfo) {
```

Comment:
variable name is lower_case with under_score.

Reply:
https://google.github.io/styleguide/cppguide.html

Reply:
updated.